### ISSUE TITLE
"OCI runtime exec failed" error

### ISSUE THREAD

**Issue by enumag** [nektos/act#456]
I'd like to use this tool to run my CI locally but I'm getting this error:

```
OCI runtime exec failed: exec failed: container_linux.go:349: starting container process caused "exec: \"bash\": executable file not found in $PATH": unknown
```

Any idea how to fix it?

**Comment by CatTheHacker** [comment]
Duplicate of #451

**Comment by enumag** [comment]
Is there a workaround?

**Comment by CatTheHacker** [comment]
@enumag see https://github.com/nektos/act/issues/451#issuecomment-752817839

**Comment by enumag** [comment]
Should be resolved by #462. Didn't have chance to try yet, I'll reopen this if there is still a problem.

**Comment by percy-raskova** [comment]
⚠️ **Missing Required Labels**
  
  This issue is missing the following required label categories:
  - **type**: Choose one of type:task, type:document, type:bug, type:feature, type:question, type:discussion
- **priority**: Choose one of priority:p0, priority:p1, priority:p2, priority:p3, priority:p4, priority:p5
- **status**: Choose one of status:todo, status:doing, status:blocked, status:review, status:done
- **time**: Choose one of time:XS, time:S, time:M, time:L, time:XL
  
  Please add the appropriate labels to this issue. For more information, see our [contribution guidelines](https://github.com/ks-sbc/KSBC/blob/main/CONTRIBUTING.md).
